website/static/play/play.js

```diff
@@ -246,6 +246,8 @@ function applyOnStatus(data) {
     }
 
     if (isLastStatus(data) || data.tour <= tour) return;
+    let audio = new Audio("bell-me.mp3");
```

Comment:
```suggestion
    let audio = new Audio("/static/play/bell-me.mp3");
```
L'URL devrait être absolue (car on a déplacé le fichier play (maintenant c'est /play))